src/components/CalendarMonth.jsx

```diff
@@ -182,7 +183,11 @@ class CalendarMonth extends React.Component {
           <strong>{monthTitle}</strong>
         </div>
 
-        <table {...css(styles.CalendarMonth_table)} role="presentation">
+        <table
+          {...css(styles.CalendarMonth_table)}
+          role="presentation"
+          aria-labelledby={`CalendarMonth_caption__${toISOMonthString(month)}`}
```

Comment:
Maybe `CalendarMonth_caption__${toISOMonthString(month)}` in a var just to make sure they both get updated together?